app/views/categories/new.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="header">
+  <%= link_to "<i class='fas fa-arrow-left'></i>".html_safe, categories_path, class: "btn btn-primary" %>
+  <h1>New Category</h1>
+  <%= form_for(current_user, url: destroy_user_session_path, method: :delete, html: { class: "nav-link" }) do |f| %>
+    <%= f.button type: :submit, class: "btn-logout" do %>
+      <%= content_tag :i, "", class: "fa-solid fa-right-from-bracket" %>
+    <% end %>
+  <% end %>
+</div>
+<div class="cat-form">
+  <%= form_for @category do |f| %>
+    <div class="field">
+      <%= f.label :name %><br>
+      <%= f.text_field :name, required: true %>
+    </div>
+    <div class="field">
+      <%= f.label :icon %><br>
+      <%= f.select :icon, { "House": "🏤", "Health": "💊", "Food": "🍕", "clothes": "👕", "School": "📚", "Gym": "🏋️‍♂️" }%>
+    </div>
+    <div class="actions">
+      <%= f.submit "Create Category" %> 
+    </div>
+  <% end %>
+</div>
+
+<style>
+  .header {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 60px;
+    padding: 8%;
+    background: #3778c2;
+    color: aliceblue;
+    gap: 3rem;
+  }
+
+  .header a, .btn{
+    color: aliceblue;
+    text-decoration: none;
+    font-weight: bold;
+  }
+
+  h1 {
+    font-size: 1.7rem;
+  }
+
+  .cat-form{
+    margin: 10%;
+    padding:10%;
+    box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+  }
+</style>
```

Comment:
- [ ] As requested by the previous reviewer, let's kindly make sure we create modular code which is the best practice in the industry. It is good practice to separate stylings. In Rails, all styles are kept in app/assets/stylesheets/application.css. Separating HTML, and CSS, makes your code more maintainable.

Reply:
@FrederickMih , I mentioned this issue in the previous request, didn't you see it? I said clearly that the external file for styling didn't work. Don't you think I don't know what are best practices?

Reply:
- [ ] @Salwa99, Unfortunately, as code reviewers we follow guidelines that are standard for all Microverse students. When you face challenges, you have to reach out to friends/code reviewers for help.

Reply:
@elisha2kyakpo1 So why didn't day suggest help or at least what should I do to fix the problem, I deal with amazing code reviewers who reached out to me and tried to help me to fix the issues in my code. So there is no excuses

Reply:
I fixed some minor styling issues and asked to run the app in the localhost instead of using the deployment link.